### PR TITLE
MomentJS fix and locale

### PR DESCRIPTION
**Pull request by ippudo**
related to #762 & #807 

Also fixed a localization issue on eventcard

### CODE REVIEW

File: src/stores/UiStore.js [hunk 1]
```diff
@@ -39,6 +62,8 @@ export default class UiStore {
     );
   }
 
+  getMomentLocale = () => moment.locale();
```
Comment (henryzbw) [inline src/stores/UiStore.js]: is it useful?

File: src/helpers/utility.js [hunk 2]
```diff
@@ -101,24 +97,6 @@ export function gasToQtum(gas) {
...
-* @return {String} Returns a formatted string.
-*/
-export function getLocalDateTimeString(unixSeconds) {
-  return moment.unix(unixSeconds).format(FORMAT_DATE_TIME);
-}
-
-export function getShortLocalDateTimeString(unixSeconds) {
-  const dateTime = moment.unix(unixSeconds);
-  if (moment.locale() === Zhlang.momentlocale) {
-    return dateTime.format(FORMAT_SHORT_DATE_TIME_CN);
```
Comment (henryzbw) [inline src/helpers/utility.js]: why not we change to 'LLL' here? so don't need to write moment.unix.  everywhere.

File: src/stores/UiStore.js [hunk 3]
```diff
@@ -29,6 +30,28 @@ export default class UiStore {
...
+    momentDurationFormat(moment);
+    moment.updateLocale('en', {
+      longDateFormat: {
+        LLL: 'M/D/YY H:mm:ss',
+        LLLL: 'MMM Do, YYYY H:mm:ss',
+      },
+    });
+    moment.updateLocale('ko', {
+      longDateFormat: {
+        LLL: 'M/D/YY H:mm:ss',
```
Comment (dwalintukan) [inline src/stores/UiStore.js]: https://www.ibm.com/support/knowledgecenter/en/SSS28S_8.1.0/XFDL/i_xfdl_r_formats_ko_KR.html

File: src/stores/UiStore.js [hunk 4]
```diff
@@ -29,6 +30,28 @@ export default class UiStore {
...
+        LLLL: 'MMM Do, YYYY H:mm:ss',
+      },
+    });
+    moment.updateLocale('zh-cn', {
+      longDateFormat: {
+        LLL: 'YY/M/D H:mm:ss',
+        LLLL: 'MMM Do, YYYY H:mm:ss',
```
Comment (dwalintukan) [inline src/stores/UiStore.js]: shouldnt chinese be YYYY, MMM Do?

**Pull request merged**